### DISCUSSION TITLE
Accessibility of lists

### DISCUSSION THREAD

**Discussion by iTrooz** [community/community#39318] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This is maybe normal since this feature is in beta and shouldn't be particularly exposed, but I find it **very** difficult to find it on the UI, both to add to a list and see my current lists

I forgot multiple times about the feature, and had to search on my search engine how to access them.

I think it should be a separate feature than stars, I don't think it serve the same purpose at all, and IIRC adding a repository to a list doesn't even star it ? (not that I would like that)

Maybe a separate tab/buttons in the profile/repository view ?

**Comment by StefanXhunga** [comment]
Thanks for your support
